src/satisfiability/planet.jl

```diff
@@ -53,14 +53,13 @@ function init_ψ(nnet::Network, bounds::Vector{Hyperrectangle})
     ψ = Vector{Vector{Int64}}()
     index = 0
     for i in 1:length(bounds)-1
-        index = set_activation_pattern!(ψ, nnet.layers[i], bounds[i], index)
+        index = set_activation_pattern!(ψ, nnet.layers[i], bounds[i+1], index)
```

Comment:
Is this i+1 the same thing as in neurify?

Reply:
I don't think I follow - which part of neurify?. I changed it to i+1  because we want it to correpsond to the layer we're indexing into. The first layer corresponds to the second index of bounds since we don't have a layer object corresponding to the input layer.

Reply:
Oh like is it from the input having bounds? I believe so